test/cmd/help.sh

```diff
@@ -6,111 +6,112 @@ set -o pipefail
-[ "$(openshift help cli 2>&1 | grep 'Developer and Administrator Client')" ]
-[ "$(openshift help kubectl 2>&1 | grep 'Kubernetes cluster')" ]
-[ "$(openshift help admin 2>&1 | grep 'Administrative Commands')" ]
+os::cmd::expect_success_and_text 'openshift help cli' 'Developer and Administrator Client'
+os::cmd::expect_success_and_text 'openshift help kubectl' 'Kubernetes cluster'
+os::cmd::expect_success_and_text 'openshift help admin' 'Administrative Commands'
 
 # help for given command with --help flag must be consistent
-[ "$(oc get --help 2>&1 | grep 'Display one or many resources')" ]
-[ "$(openshift cli get --help 2>&1 | grep 'Display one or many resources')" ]
-[ "$(openshift kubectl get --help 2>&1 | grep 'Display one or many resources')" ]
-[ "$(openshift start --help 2>&1 | grep 'Start an all-in-one server')" ]
-[ "$(openshift start master --help 2>&1 | grep 'Start a master')" ]
-[ "$(openshift start node --help 2>&1 | grep 'Start a node')" ]
-[ "$(oc get --help 2>&1 | grep 'oc')" ]
+os::cmd::expect_success_and_text 'oc get --help' 'Display one or many resources'
+os::cmd::expect_success_and_text 'openshift cli get --help' 'Display one or many resources'
+os::cmd::expect_success_and_text 'openshift kubectl get --help' 'Display one or many resources'
+os::cmd::expect_success_and_text 'openshift start --help' 'Start an all-in-one server'
+os::cmd::expect_success_and_text 'openshift start master --help' 'Start a master'
+os::cmd::expect_success_and_text 'openshift start node --help' 'Start a node'
+os::cmd::expect_success_and_text 'oc get --help' 'oc'
 
 # help for given command through help command must be consistent
-[ "$(oc help get 2>&1 | grep 'Display one or many resources')" ]
-[ "$(openshift help cli get 2>&1 | grep 'Display one or many resources')" ]
-[ "$(openshift help kubectl get 2>&1 | grep 'Display one or many resources')" ]
-[ "$(openshift help start 2>&1 | grep 'Start an all-in-one server')" ]
-[ "$(openshift help start master 2>&1 | grep 'Start a master')" ]
-[ "$(openshift help start node 2>&1 | grep 'Start a node')" ]
-[ "$(openshift cli help update 2>&1 | grep 'openshift')" ]
-[ "$(openshift cli help replace 2>&1 | grep 'openshift')" ]
-[ "$(openshift cli help patch 2>&1 | grep 'openshift')" ]
+os::cmd::expect_success_and_text 'oc help get' 'Display one or many resources'
+os::cmd::expect_success_and_text 'openshift help cli get' 'Display one or many resources'
+os::cmd::expect_success_and_text 'openshift help kubectl get' 'Display one or many resources'
+os::cmd::expect_success_and_text 'openshift help start' 'Start an all-in-one server'
+os::cmd::expect_success_and_text 'openshift help start master' 'Start a master'
+os::cmd::expect_success_and_text 'openshift help start node' 'Start a node'
+os::cmd::expect_failure_and_text 'openshift cli help update' 'openshift'
+os::cmd::expect_failure_and_text 'openshift cli help replace' 'openshift'
+os::cmd::expect_failure_and_text 'openshift cli help patch' 'openshift'
```

Comment:
The three expressions here:
```
openshift cli help update
openshift cli help replace
openshift cli help patch
```

Are not valid commands - their error output is:
```
error: unknown command "help replace"
See 'openshift cli -h' for help and examples.
```

What is their purpose?